respy/simulate.py

```diff
@@ -321,6 +373,7 @@ def _simulate_single_period(
     See docs for more information!
 
     """
+    choice_set = complex_tuple[1]
```

Comment:
Maybe we should write a function that retrieves information from complex. This hardcoding in an abstract function does someho look a bit weird.

Reply:
Done!